### PR TITLE
[GCMSLG-103] Update header search field on mobile.

### PR DESCRIPTION
**Pull request by alan-cole**
* Search field to sit below logo on mobile (0 - 576px).
* Search label to read 'Enter search terms'.
* Search submit to read 'Submit search'.
* Search submit to use icon.

screen shot 2018-11-26 at 3 27 09 pm
screen shot 2018-11-26 at 3 26 59 pm
screen shot 2018-11-26 at 3 26 49 pm
screen shot 2018-11-26 at 3 26 41 pm

### CODE REVIEW

File: govcms8_uikit_starter.theme [hunk 1]
```diff
@@ -113,6 +113,13 @@ function govcms8_uikit_starter_preprocess_page(&$variables) {
   }
 }
 
+function govcms8_uikit_starter_form_alter(&$form, FormStateInterface $form_state) {
```
Comment (gargsuchi) [inline govcms8_uikit_starter.theme]: @alan-cole  Please add 
/**
 * Implements hook_form_alter().
 */

File: package-lock.json [hunk 2]
```diff
@@ -19,9 +19,9 @@
       }
     },
     "@gov.au/animate": {
-      "version": "1.0.9",
-      "resolved": "https://registry.npmjs.org/@gov.au/animate/-/animate-1.0.9.tgz",
-      "integrity": "sha512-KL+k1jprAMbXxYp0RtCp9ETZ523wf+BIR6F48yUNggrIavxfMvpQjyoBwGhxXnPc5wxSQGZkNCD0NWwlLWrFHA==",
+      "version": "1.0.10",
```
Comment (gargsuchi) [inline package-lock.json]: @alan-cole Are these changes in scope of the ticket?

**Review by gargsuchi — CHANGES_REQUESTED** [review]

File: govcms8_uikit_starter.theme (hunk 1 above)
Comment (gargsuchi) [reply govcms8_uikit_starter.theme]: Also, you should check the form_id.

File: package-lock.json (hunk 2 above)
Comment (alan-cole) [reply package-lock.json]: Running an `npm install` in order to compile the styles will auto-bump any minor changes to npm packages and store that in the package-lock. The styles.css file I commit as part of this ticket is compiled with whatever versions are in the package-lock. 

Generally (in other projects) these package-lock versions would be committed as they occur, but if you'd rather have these committed separately we could do that too. I think it may just run a chance that if a package version difference causes the compiled files to render differently, it may cause some confusion.

Comment (alan-cole) [reply package-lock.json]: @tobybellwood Do you have any opinion on this?

**Review by gargsuchi — APPROVED** [review]

**Comment by gargsuchi** [comment]
Approved - but @tobybellwood there is a question for you above.

File: package-lock.json (hunk 2 above)
Comment (tobybellwood) [reply package-lock.json]: Given we'll adopt a merge-to-branch strategy for these A11y fixes - we'll probably end up with another final package-lock update at the end - there is always the risk that the @latest module doesn't affect the target of the PR, but may have an adverse effect somewhere else.  I'd suggest that unless the package-lock contains an update to a package that is critical to the PR being raised, it can probably be omitted?
